.bluemix/pipeline.yml

```diff
@@ -29,24 +29,28 @@ stages:
 
       # List the conversation services, for example:
       # $ cf services | grep conversation
-      # Chatbot-20170822164726835   conversation          free                create succeeded
-      # Conversation-hf             conversation          free                create succeeded
-      # NOTE: the -c option for grep counts number of rows
-      result=$(cf services | grep -c conversation)
+      # Chatbot-20170822164726835   conversation    free    create succeeded
+      # Conversation-hf conversation    free    create succeeded
+      BAE_EXISTS=$(cf services | grep conversation | grep BAE)
```

Comment:
for some reason the script it bombing out over here and not going forward with the echo, any ideas?

```bash
+ CHATBOT_JSON_URL=https://developer.ibm.com/exchange/wp-content/uploads/sites/144/2017/07/appetito.json
+ CHATBOT_NAME=Appetito,%20your%20virtual%20waiter
+ curl -L -o workspace.json https://developer.ibm.com/exchange/wp-content/uploads/sites/144/2017/07/appetito.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  9489  100  9489    0     0   3989      0  0:00:02  0:00:02 --:--:--  3988
++ cf services
++ grep conversation
++ grep BAE
+ BAE_EXISTS=
```

Reply:
It's not an authentication issue, if I add the 'cf services' command to line 33, just before the grep, it works just fine:

```
smartinelli-mac chatbot-deployer $ ./basho.sh 
+ CHATBOT_JSON_URL=https://developer.ibm.com/exchange/wp-content/uploads/sites/144/2017/07/appetito.json
+ CHATBOT_NAME=Appetito,%20your%20virtual%20waiter
+ curl -L -o workspace.json https://developer.ibm.com/exchange/wp-content/uploads/sites/144/2017/07/appetito.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  9489  100  9489    0     0  59152      0 --:--:-- --:--:-- --:--:-- 59306
+ cf services
Getting services in org stevemar@ca.ibm.com / space dev as stevemar@ca.ibm.com...
OK

name                  service               plan   bound apps   last operation
Continuous Delivery   continuous-delivery   Free                create succeeded
storage               Object-Storage        Free                create succeeded
++ cf services
++ grep conversation
++ grep BAE
+ BAE_EXISTS=
```

Reply:
Removing the "e" from "#!/bin/bash -ex" fixed this